inc/namespace.php

```diff
@@ -75,4 +75,31 @@ function bootstrap() {
 		define( 'ALTIS_ANALYTICS_PINPOINT_ENDPOINT', getenv( 'ALTIS_ANALYTICS_PINPOINT_ENDPOINT' ) );
 		define( 'ALTIS_ANALYTICS_COGNITO_ENDPOINT', getenv( 'ALTIS_ANALYTICS_COGNITO_ENDPOINT' ) );
 	}
+
+	add_filter( 'qm/output/file_path_map', __NAMESPACE__ . '\\set_file_path_map', 1 );
+}
+
+/**
+ * Enables Query Monitor to map paths to their original values on the host.
+ *
+ * @param array $map Map of guest path => host path
+ * @return array Adjusted mapping of folders
+ */
+function set_file_path_map( array $map ) : array {
+	if ( ! file_exists( '/etc/chassis-constants' ) ) {
```

Comment:
Is this meant to be in the local chassis repo? This file won't exist on local server

Reply:
Odd. I figured out an alternative method that works at least. Cheers for getting this off the ground. It’s very cool